readme.md

```diff
@@ -137,6 +139,24 @@ You may see the following error when cloning the repo in your Mac terminal: `xcr
 
 If your GitHub account has two-factor authentication enabled, you will need to create an access token to log into your account in the terminal. Go to [github.com/settings/profile](https://github.com/settings/profile), click **Developer Settings**, then **Personal access tokens** and generate a new token. This will be your password when signing in to GitHub in the terminal.
 
+### Getting a CORS error when requesting a unicorn
+
+There are a couple of possible causes of CORS errors in this app. One would be not enabling CORS in the API Gateway, which is covered in the [Add a Rest API resource step](https://github.com/stackery/wild-rydes-workshop/blob/master/04-backend-api.md#7-update-requestunicorn-function).
+
+The other is forgetting to copy the new function code for the `RequestUnicorn` function, which is covered in the [Update RequestUnicorn Function step](https://github.com/stackery/wild-rydes-workshop/blob/master/04-backend-api.md#7-update-requestunicorn-function).
+
+If you have completed both steps and are still getting a CORS error in the console, please notify a team member and we'll help you debug.
+
+### "CloudFormation cannot update a stack when a custom-named resource request"
```

Comment:
How did this occur?

Reply:
I can't remember who triggered that, but it was an error I saw towards the end of the workshop.

Reply:
I think they were likely doing something wrong, but in a way that won't be generically helpful to describe in the troubleshooting docs shown here. I think we should remove this section.